packages/grid/_modules_/grid/components/cell/GridCell.tsx

```diff
@@ -59,9 +59,6 @@ export const GridCell: React.FC<GridCellProps> = React.memo((props) => {
   const valueToRender = formattedValue || value;
   const cellRef = React.useRef<HTMLDivElement>(null);
   const apiRef = React.useContext(GridApiContext);
-  const currentFocusedCell = apiRef!.current.getState().keyboard.cell;
-  const isCellFocused =
```

Comment:
Why did you remove that check? It is there to make sure that only the cell with coordinates `0-0` has `tabIndex={0}`.

Reply:
yes I understand but I'm not it is correct as mentioned in my other comment

Reply:
Ok, I'll resolve that one in favour of the other one to keep the discussion in a single place.